server/internal/adapter/gql/resolver_request.go

```diff
@@ -25,13 +25,35 @@ func (r requestResolver) Project(ctx context.Context, obj *gqlmodel.Request) (*g
 }
 
 func (r requestResolver) Reviewers(ctx context.Context, obj *gqlmodel.Request) ([]*gqlmodel.User, error) {
-	res, err := dataloaders(ctx).User.LoadAll(obj.ReviewersID)
-	if len(err) > 0 {
-		return nil, err[0]
-	}
-	return res, nil
+	res, errors := dataloaders(ctx).User.LoadAll(obj.ReviewersID)
+	return res, errors[0]
```

Comment:
Unnecessary change: if errors is nil or empty, it panics

Reply:
`errors` always [nil]

Reply:
Future implementation changes may cause an error to be returned. There is no particular harm in handling errors, so they should be left as they are.